OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -522,6 +529,7 @@ static bool CanCreateGLWindow(GLProfile profile)
 			var window = SDL.SDL_CreateWindow("", 0, 0, 1, 1, flags);
 			if (window == IntPtr.Zero || !string.IsNullOrEmpty(SDL.SDL_GetError()))
 			{
+				errorLog.Add(profile, SDL.SDL_GetError());
```

Comment:
Shouldn't we be logging from `SDL_Init` too? We could use a standard list and prefix the error message with ([profile]).

Reply:
I don't really have a test case for `SDL_INIT`. That was mostly inspired by a StyleCop warning.

Reply:
This leaves an edge case where we can have no supported profiles but no error message.